openedx/core/djangoapps/user_api/accounts/tests/test_filters.py

```diff
@@ -0,0 +1,126 @@
+"""
+Test that various filters are fired for views in the certificates app.
+"""
+from django.http import HttpResponse
+from django.test import override_settings
+from openedx_filters import PipelineStep
+from openedx_filters.learning.filters import AccountSettingsRenderStarted
+from rest_framework import status
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+
+
```

Comment:
Check [this](https://github.com/openedx/edx-platform/blob/8b1c90a19d395bafbc2d76cbc35e1056f97df763/lms/djangoapps/courseware/tests/test_filters.py#L16) test implementation. You're missing one exception.